content/questions/admin/question-049.md

```diff
@@ -0,0 +1,12 @@
+---
+question: "What can a member of a GitHub organization do by default?"
+archetype: "questions"
+title: "Question 049"
+draft: false
+---
+
+> https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization
+1. [x] They can create private repositories (if allowed by the organization's settings) and collaborate on any project within the organization.
```

Comment:
`and collaborate on any project within the organization.`

Are you sure this is true? If you are invited to an organization you still need to be added to a repository or a team that is invited to a repository.

You dont get access to all repositories by being a member